avm/res/network/vpn-server-configuration/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,84 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+// e.g., for a module 'network/private-endpoint' you could use 'dep-dev-network.privateendpoints-${serviceShort}-rg'
```

Comment:
Great work @ericscheffler, just a small ask to please remove these tutorial comments from all test templates :)

Reply:
<sigh> I guess....

Also, question about line #199 in the main.bicep file I marked the user-defined type as exportable (I plagiarized that from another unrelated resource example); while I'm at it should I remove this as well?

Reply:
No, the `export()` is fine. We may actually do this for all UDTs in the `main.bicep` templates eventually to enable users to import & re-use those types for their own parameters 💪